test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptionsNoCache_test.cpp

```diff
@@ -473,9 +512,130 @@ TEST(mongoUpdateContext_withOnchangeSubscriptionsNoCache, Cond1_appendMatch)
     CHECK_LAST_NOTIFICATION("51307b66f481db11bf860001", 1360232700);
 
     delete notifierMock;
-    delete timerMock;
+
+    utExit();
+}
+
+
+
+/* ****************************************************************************
+*
+* Cond1_appendMatch_typePattern -
+*/
+TEST(mongoUpdateContext_withOnchangeSubscriptionsNoCache, Cond1_appendMatch_typePattern)
```

Comment:
This test and the next one worth the time to be reviewed.

Note that two new subscriptions (sub10 and sub11) has been populated in database to make these test to work (see prepareDatabase() function).